pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --reuse-db --tb=native
+addopts = --reuse-db
```

Comment:
I don't know why I put this here in the first place... it's completely stupid, it removes the nice tracebacks/test failures display that is one of the biggest advantage of using pytest.